portability-types-common/src/main/java/org/datatransferproject/types/common/models/photos/PhotoModel.java

```diff
@@ -22,8 +22,11 @@
 
```

Comment:
(same) skipping review as I think this belongs to #1079 (feel free to merge in either - but I only reviewed the other copy)